changelog.d/14361.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in #14228 where the power level event was incorrectly created during initial room creation.
```

Comment:
was the bug ever released?
If so, cite the version number instead.
If not, probably just coalesce the changelogs.

Reply:
Yes, it was released in most recent rc. I've updated the changelog.

Reply:
Ah, is it serious enough to go into the release branch?
Actually it occurs to me that I don't understand what 'the power level event was incorrectly created' means — at first thought this sounds like the power levels event wasn't meant to be created, but surely that can't be right.

Iit would be best to note the user-visible bug if there is one. I honestly have lost track of this over the weekend so I'm afraid I can't remember enough to advise.

Reply:
There isn't an end-user facing bug afaict. From my understanding, the worst thing that could happen with this bug is the code could break unexpectedly if other changes were made to the surrounding code, so mostly fixing this is to a: prevent confusion in the future and b: set up the code for further changes (such as adding the membership event to the batch or batching up state groups).